src/components/shared/Description.tsx

```diff
@@ -1,8 +1,13 @@
 import { MarkdownViewer } from '@stoplight/markdown-viewer';
+import { Box } from '@stoplight/mosaic';
 import * as React from 'react';
 
 export const Description: React.FunctionComponent<{ value: string }> = ({ value }) => (
-  <div className="sl-w-full sl-text-muted" title={value}>
-    <MarkdownViewer markdown={value} />
-  </div>
+  <Box
+    as={MarkdownViewer}
+    markdown={value}
+    style={{
+      fontSize: 12,
+    }}
+  />
```

Comment:
I was trying to avoid the extra containing element, and we can't use `fontSize='sm'` because of css specificity stuff. This is a bit unique in that the way it's working is that `.sl-prose` styling (which is what `MarkdownViewer` is using under the hood) uses `em` units for most everything, at a base font size of `16px`. This allows us to scale the entire markdown viewer styling down by just adjusting the root font size on the `.sl-prose` class, but we can't do that with another root css class (`.sl-text-sm`) - need to do that with a style attribute so that it has higher specificity. Does that make sense?

The final thing is that didn't expose a `style` attribute on `MarkdownViewer` props, so the only way to get that right now is to compose with `Box` (or wrap in extra div that has a style prop). Come to think of it, this use case re adjusting the overall size of markdown contents might be important enough to justify its own `size` or `fontSize` prop on `MarkdownViewer` (that under the hood sets `style.fontSize`) - what do you think?

![2021-06-02 14 46 40](https://user-images.githubusercontent.com/847542/120535845-125e6e80-c3a9-11eb-99b8-04a07c1bd45a.gif)

Reply:
Hmm... I *think* I understand it, thanks for the explanation. Yeah a `fontSize` prop might be warranted but I see what's going on here.

Reply:
Alright I'll try to get a fontSize prop into markdown-viewer before we move it out of beta - issue here: https://github.com/stoplightio/markdown-viewer/issues/94. Won't block this PR though.